build.gradle

```diff
@@ -0,0 +1,78 @@
+plugins {
+    id 'org.springframework.boot' version '2.5.6'
+    id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+    id 'jacoco'
+    id 'java'
+}
+
+group = 'ru.hse'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '11'
+
+configurations {
+    compileOnly {
+        extendsFrom annotationProcessor
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'org.springframework.boot:spring-boot-starter'
+    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+    compileOnly 'org.projectlombok:lombok'
+    annotationProcessor 'org.projectlombok:lombok'
+    testImplementation 'org.springframework.boot:spring-boot-starter-test'
+    implementation "com.google.code.gson:gson:2.8.9"
+    implementation "org.jacoco:org.jacoco.core:0.8.7"
```

Comment:
Why do you need jacoco inside the application? Does application use it somehow? Otherwise, it is really bad practice to bundle code-coverage library into application.

Reply:
Fixed in #5 PR